README.md

```diff
@@ -37,67 +21,220 @@ Hello
 
 # 📖 [Hello-Microverse] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+<a name="readme-top"></a>
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+
+  <h3><b>My Hello Microverse project</b></h3>
+
```

Comment:
- [ ] Well done on using the `README` template, it makes your repository look professional and well-documented. However, I noticed that there is some unnecessary text in your` README.` As you already have a table above, this additional text is not required. Would you mind removing it? 🎯